### ISSUE TITLE
bug(COMPONENT): Sort Header

### ISSUE THREAD

**Issue by smil-thakur** [angular/components#30196]
### Is this a regression?

- [x] Yes, this behavior used to work in the previous version

### The previous version in which this bug was not present was

v18

### Description

When focusing on sort header using "tab" key previously on version 18 the sort headers were visible with less opacity but on version 19 the header icons are not visible

old;:
![Image](https://github.com/user-attachments/assets/a430931f-32d9-4327-a772-d9f8676c6578)

new:
![Image](https://github.com/user-attachments/assets/b0b26a2c-8fd0-4c64-b653-160396472beb)

as you can see on new one the header icons not visible on focus

### Reproduction

https://material.angular.io/components/sort/overview

check over here and compare with

https://v18.material.angular.io/components/sort/overview

### Expected Behavior

expected behavior should be sort icon should be visible with some opacity on focus and on click or enter press it should be visible with full opacity

### Actual Behavior

sort icon is not at all visible on focus

### Environment

- Angular: 19
- CDK/Material: 19
- Browser(s): chrome
- Operating System (e.g. Windows, macOS, Ubuntu): windows

**Comment by smil-thakur** [comment]
@amysorto sorry for pinging, if want to look into this problem how can is test it locally, like i cloned the repo, installed packages through yarn install, but was not able to run something to test it locally, can you guide me or share some docs for runinng and testing material components locally and fixing problems

**Comment by amysorto** [comment]
If you are interested in looking into contributing a fix, here are instructions for our repo to set up your dev environment: https://github.com/angular/components/blob/main/DEV_ENVIRONMENT.md. The dev app has examples for table once it's run locally which you can test with.
